File: 프로그래머스/0/181888. n개 간격의 원소들/n개 간격의 원소들.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* solution(int num_list[], size_t num_list_len, int n) {
    
    
    int* answer = (int*)malloc(sizeof(int)*num_list_len);
    int a=0;
    for(int i=0; i<num_list_len; i+=n, a++)
    {
        answer[a]=num_list[i];
    }
    return answer;
}

File: 프로그래머스/0/181888. n개 간격의 원소들/n개 간격의 원소들.js
function solution(num_list, n) {
    
    let answer = [];
    
    let a = 0;
    for(let i = 0; i < num_list.length; i += n) {
        answer[a] = num_list[i];
        a++;
    }
    
    return answer;
}